### PR TITLE
#236 new class e2e tests + #233 classes screen e2e tests

### LINKED ISSUE #233: classes screen e2e tests (see description)
* teacher
        * should see only it’s own class
        * should not be able to add class
    * principle
        * should see all classes
        * can navigate to add class
    * general
        * can navigate to edit class
        * can remove class

child of #228

### LINKED ISSUE #236: new class e2e tests (see description)
* should be able to create new class with name and grade
    * should inherit default schedule for grades A-C and other schedule D-F (consider separating to two e2e tests
    * schedule should be enabled for created classes only

child of #228

### PR DESCRIPTION
**Pull request by itziklid**
closes #233 
closes #236

### CODE REVIEW

File: src/app/pages/class-details/class-details.view/class-details.view.component.html [hunk 1]
```diff
@@ -7,16 +7,16 @@
...
                 required>
               <i class="material-icons" matSuffix>mode_edit</i>
             </mat-form-field>
           </div>
 
           <div class="class-detail-field">
             <mat-form-field class="class-detail-field">
-              <mat-select [(value)]="grade" placeholder="שכבה" (selectionChange)="detailChanged.emit({name: name, grade: grade})">
-                <mat-option *ngFor="let key of gradesKeys" [value]="key" selected="key === grade">
+              <mat-select [attr.data-test-id]="'grade-select'" [(value)]="grade" placeholder="שכבה" (selectionChange)="detailChanged.emit({name: name, grade: grade})">
```
Comment (dimkinv) [inline src/app/pages/class-details/class-details.view/class-details.view.component.html]: hey, is the binding needed? can we just use the `data-` attribute?

File: tests/pageobjects/navbar.po.ts [hunk 2]
```diff
@@ -29,22 +29,26 @@ export default class NavbarPage {
...
-    await t.click(this.menuDropDown).click(this.menuDropDownLessons);
+    await this.navigateTo(this.menuDropDownLessons);
   }
 
   async navigateToClassSchedulePage() {
-    await t.click(this.menuDropDown).click(this.menuDropDownClassSchedule);
+    await this.navigateTo(this.menuDropDownClassSchedule);
+  }
+
+  async navigateTo(pageSelector: Selector) {
```
Comment (dimkinv) [inline tests/pageobjects/navbar.po.ts]: this one can be `private` or we use it outside of this file?

File: src/app/pages/class-details/class-details.view/class-details.view.component.html (hunk 1 above)
Comment (itziklid) [reply src/app/pages/class-details/class-details.view/class-details.view.component.html]: Done.

File: tests/pageobjects/navbar.po.ts (hunk 2 above)
Comment (itziklid) [reply tests/pageobjects/navbar.po.ts]: changed to private.

**Pull request merged**